source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,181 @@
+# Install Prerequisites
+
+[//]: # (If any commented statements become false, change prior text)
+
+Before you begin, you will need to have access to our Beta. If you don't have access yet, request access at [bons.ai][1] and then come visit us in the [Forums][17] and let us know why you're excited to try out Bonsai!
+
+The next section walks you through how to install Python, Git, and Pip, but if you're already ahead of the curve and have them all installed, skip down to [Setup the Bonsai CLI][2]. 
+
+**Note:** We highly recommend Mac users use [HomeBrew][6] or download Python directly in the General Install instructions so they can perform the following steps on user-land Python rather than [system Python][13].
+
+**If you have Anaconda installed on your system, skip down to [Install with Anaconda][9].**
+
+If you *do not* have Anaconda installed on your system continue to follow our **General Install** instructions.
+
+## General Install
+
+### Windows
+
+> [Chocolatey][14] commands
+
+```shell
+choco install git
+choco install python # or you can install python2 if you need version 2.7
+```
+
+1. **Git**: Download and install Git from [Git for Windows][11] or with [Chocolatey][14].
+2. **Python**: Download and install Python from [python.org][3] or with [Chocolatey][14].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+**Note:** You may need to reboot your computer after installing Python and Git to make sure all install settings take effect.
+
+### Mac OS X
+
+> [Homebrew][6] commands
+
+```shell
+brew install git
+brew install python3 # or you can install python if you need version 2.7
+```
+
+1. **Git**: Git is more than likely already installed on your computer if you have Xcode, but if not, run the git command or install with [Homebrew][6].
+2. **Python**: Download and install Python from [python.org][3] or with [Homebrew][6].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+## Install with Anaconda
+
+If you *do not* have Anaconda installed on your system, please skip this section.
+
+conda is a tool for installing and managing Python and R dependencies. For more information, refer to the [Anaconda website][10].
```

Comment:
Anaconda only covers python and pip.  Anaconda was invented by a company named Continuum Analytics before pip and virtualenv were common place (maybe even existed). Anaconda's goal was to fix the python install process. Like many new standards it's now yet another fork in how python gets installed. This makes Anaconda kind of like an alternative python universe in terms of install experience.  You can make Anaconda install pip though and therefore use both install experiences.